### ISSUE TITLE
USER ISSUE: Kicked for suspect speed cheat

### ISSUE THREAD

**Issue by VeryBlindNinja** [StrangeLoopGames/EcoIssues#21611]
\
**User Id: (slg117490)**
**Version:** 0.9.3.2 beta release-217

**Steps to Reproduce:**
Entered Steam tracktor, kick for suspected movement cheating
Run while jumping, kicked for same reason
I am not speed cheating

**Expected behavior:**
Not getting kicked for entering my steam truck

**Actual behavior:**
The following error message popped up:
```
Cheat

<size=60.00%>Cheat movement detected. Report the issue if you're not cheating.</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
The server has a QOL big shovel mod. Nothing I installed and cannot uninstall since it's on the server's end.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by VeryBlindNinja** [comment]
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6388521/Player.log)

**Comment by SlayksWood** [comment]
@VeryBlindNinja Thank you for the report. merged to #21609
